nats-base-client/util.ts

```diff
@@ -34,13 +34,13 @@ export function protoLen(ba: Uint8Array): number {
       return n + 1;
     }
   }
-  return -1;
+  return 0;
 }
 
 export function extractProtocolMessage(a: Uint8Array): string {
   // protocol messages are ascii, so Uint8Array
   const len = protoLen(a);
-  if (len) {
+  if (len > 0) {
```

Comment:
Since you changed from -1 to 0, the test `if (len) {` would then become valid, but I am ok being explicit here.

Reply:
yeap going to keep it that way - at some point I believe the code returned 0, and then changed (this was used by the protocol parser before. Testing for what we are looking for is always a good thing.